reference-adapter/ifix-department-entity-service/src/main/resources/db/seed/V20210824192300__department_hierarchy_seed_data

```diff
@@ -1,4 +1,4 @@
-db.departmentHierarchyLevel.insertMany([
+db.departmentHierarchyLevelDTO.insertMany([
```

Comment:
I think this is MongoDB-specific code. Remove it from the project.

Reply:
Yes it is, Kept for future reference to support old project seed data.
I'll remove it.